platform_versioned_docs/version-24.1/enterprise/kubernetes-minimal-install.mdx

```diff
@@ -0,0 +1,418 @@
+
+        ```
+        kubectl apply -f tower-svc.yml
+        ```
+
+        <details>
+        <summary>View command output</summary>
+        ```
+        deployment.apps/backend configured
+        deployment.apps/frontend configured
+        service/backend configured
+        service/frontend configured
+        ```
+        </details>
+
+    1. Confirm that cron service deployed successfully:
+
+        ```
+        kubectl get deployments
+        ```
+
+        <details>
+        <summary>View command output</summary>
+        ```
+        NAME       READY   UP-TO-DATE   AVAILABLE   AGE
+        backend    1/1     1            1           2d6h
+        cron       1/1     1            1           2d6h
+        frontend   1/1     1            1           2d6h
+        ```
+        </details>
+
+## 7. Create a new user account
+
+Use the same email address that you specified for the `TOWER_ROOT_USERS` environment variable.
+
+1. Open a port forward to the frontend to access the Seqera UI:
+
+    ```
+    kubectl port-forward services/frontend 8080:80 &
+    ```
+
+    <details>
+    <summary>View command output</summary>
+    Forwarding from 127.0.0.1:8080 -> 8000
+    Forwarding from [::1]:8080 -> 8000
+    </details>
+
+1. In a web browser, visit `http://localhost:8080/`. In the **Sign in to Seqera platform** form, enter the email address that you set the `TOWER_ROOT_USERS` environment variable to.
+
+1. Create a file named `mailcatcher.yml` with the following manifest:
+
+    <details>
+    <summary>Show `mailcatcher.yml` file</summary>
+    ```js file=./_templates/k8s/k8s_min/mailcatcher.yml showLineNumbers
+    ```
+    </details>
+
+1. Deploy the MatchCatcher application, so that you can access the email that Seqera sends with your authentication token:
+
+    ```
+    kubectl apply -f mailcatcher.yml
+    ```
+
+1. Open a port forward to the MailCatcher application, to access its web UI:
+
+    ```
+    kubectl port-forward services/mailcatcher 1080 &
+    ```
+
+    <details>
+    <summary>View command output</summary>
+    ```
+    Forwarding from 127.0.0.1:1080 -> 1080
+    Forwarding from [::1]:1080 -> 1080
+    ```
+    </details>
+
+1. In a web browser, visit `http://localhost:1080/`.
+
+1. Open the authentication email with the subject _Complete your sign-in to Seqera_ and select **Complete sign-in**. Seqera redirects your browser to the **Launchpad** page.
+
+1. Optional: To stop the background port forward processes, you can run `jobs` in your shell, and then run `kill -15 <pid>` to stop each process. If you stop the port forward to the frontend, you can no longer access the Seqera UI. We recommend using a cloud provider's load balancers in conjunction with the Kubernetes ingress functionality for persistent access to your installation.
+
+Your installation of Seqera Platform is complete.
+
+## Next steps
+
+- Configure OpenID Connect (OIDC) for seamless integration with your identify provider
```

Comment:
```suggestion
- Configure OpenID Connect (OIDC) for seamless integration with your identity provider
```

What about `Configure identity providers for seamless single sign-on` we support more than OIDC AFAIK.